### ISSUE TITLE
Populate a11y.js URLs dynamically from scenario data 

### ISSUE THREAD

**Issue by radiocontrolled** [bbc/simorgh#945]
**Is your feature request related to a problem? Please describe.**
Currently URLs to test with bbc-ally are manually added in` simorgh/a11y.js` (see https://github.com/BBC-News/simorgh/pull/876/files). 

They should be populated by our scenario data instead. We should run all our scenarios through bbc-a11y eventually.

**Describe the solution you'd like**
Adapt a11y.js config, e.g. making use of `fs.readdir('data/prod/news/')` to pull in some or all of our scenarios. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

**Comment by chris-hinds** [comment]
Refining this issue with the Core POD we believe we should not be testing against every fixture data asset with have locally and instead produce a `worst case scenario` asset. E.g an optimo article with every component, for a rtl and ltr language.

**Comment by jamesbhobbs** [comment]
Superseded by https://github.com/bbc/simorgh/issues/2221
